experimental/generation/generator/packages/library/src/dialogGenerator.ts

```diff
@@ -995,7 +995,7 @@ export async function generate(
 
         if (!error) {
             // Write final schema
-            const body = stringify(expanded, (key: any, val: any) => (key === '$templates' || key === '$requires' || key === '$templateDirs' || key === '$examples' || key === '$template' || key === '$generator') ? undefined : val)
```

Comment:
>key === '$examples'  [](start = 144, length = 20)

Why did you have to add this back?  $examples are just for the generation process so we should not need it in the final schema.